### PR TITLE
fixed typo and metagenomic QCSummary message

### PR DESCRIPTION
**Pull request by mattheww95**
Updated the QCMessage for mikrokondo metagenomic runs and adding a QCSummary and a QCMessage

### CODE REVIEW

File: modules/local/report.nf [hunk 1]
```diff
@@ -245,18 +245,21 @@ def create_action_call(sample_data, species_tag){
                 // nothing to do here
                 if(params.metagenomic_run){
                     final_message = "No QC Summary is provided for metagenomic samples."
+                    qc_summary = "No quality control criteria is applied for metagenomic samples."
```
Comment (apetkau) [inline modules/local/report.nf:248]: Could you add test cases for this? E.g., in this case adding a case to set `params.metagenomic_run` and make sure this message is returned.

Do you have any small data (or ways to update small datasets) so it forces detection of metagenomic samples?

**Review by apetkau — COMMENTED** [review]
This looks great. Thanks @mattheww95 . I just have one comment/question

File: modules/local/report.nf (hunk 1 above)
Comment (mattheww95) [reply modules/local/report.nf:248]: I can make something small! I may be able to spoof some inputs.

Comment (mattheww95) [reply modules/local/report.nf:247]: fixed in: https://github.com/phac-nml/mikrokondo/pull/103/commits/3b07a9be11a0aff6928d8269e4eb17a08e63d496

**Review by apetkau — APPROVED** [review]
Looks great to me. Thanks so much Matthew for creating additional datasets and adding those test cases 😄

**Pull request merged**